reform-db/cmd_init_sqlite3.go

```diff
@@ -82,7 +82,9 @@ func initModelsSQLite3(db *reform.DB) (structs []StructData) {
 
 		rows, err := db.Query("PRAGMA table_info(" + tableName + ")") // no placeholders for PRAGMA
 		if err != nil {
-			logger.Fatalf("%s", err)
+			// https://github.com/go-reform/reform/issues/180
+			logger.Printf("Skipping %s: %s.", tableName, err)
```

Comment:
Ok, I can't review these changes because the whole code of this function confuses me a lot... cyclomatic complexity makes it unreadable to me, sorry :/ 

For example, on the line 108 you check an error... Where is this error from? Is it from line 91? Or is it from the line 83?.. Quite unclear to me from the first sight...

The change itself is probably ok, it doesn't seem to make the code worse, but the function is not simple. If you are interested, I'd offer a pair programming session to maybe refactor it (as a part of some other issue / PR).

Reply:
P.S. Maybe it might be helpful to add a test to cover this case - when one of the tables gets error here...